cmd/crossplane/core/core.go

```diff
@@ -403,16 +404,25 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 		return errors.Wrap(err, "cannot setup API extension controllers")
 	}
 
-	var pr pkgcontroller.PackageRuntime
-	switch c.PackageRuntime {
-	case string(pkgcontroller.PackageRuntimeDeployment):
-		pr = pkgcontroller.PackageRuntimeDeployment
-	case string(pkgcontroller.PackageRuntimeExternal):
-		pr = pkgcontroller.PackageRuntimeExternal
+	var pr pkgcontroller.ActiveRuntime
+	switch rt := pkgcontroller.PackageRuntime(c.PackageRuntime); rt {
```

Comment:
It would be good to document the expected format for this flag in the `help` string.

Reply:
Done.